CONTRIBUTING.adoc

```diff
@@ -145,48 +145,46 @@ TIP: To ensure you get the latest image first run `vagrant box remove fedora_ins
 
         $ vagrant ssh
 
-7.  Run a build:
+7.  You are now ready to edit the source, rebuild and restart OpenShift to test your changes.
+
+8.  Run a build:
 
         $ cd /data/src/github.com/openshift/origin
         $ make clean build
 
-8.  You are now ready to edit the source, rebuild and restart OpenShift to test your changes.  At this point you may want to update your $PATH:
-
-        # back to /home/vagrant
-        $ cd
-        # update path to include binaries for oc, oadm, etc
-        # this is temporary, to make it persistent add it to .bash_profile
-        $ export PATH=/data/src/github.com/openshift/origin/_output/local/bin/linux/amd64:$PATH
-
-
 9.  Now start the OpenShift server:
 
-        # redirect the logs to  /home/vagrant/openshift.log for easier debugging
-        $ sudo `which openshift` start --public-master=localhost &> openshift.log &
+        $ sudo service openshift start
+
+    Or:
+    
+        # must cd / to use prepopulated $KUBECONFIG
+        $ cd /
+        # redirect the logs to /home/vagrant/openshift.log for easier debugging
+        $ sudo `which openshift` start --public-master=localhost &> $HOME/openshift.log &
```

Comment:
Yes, I believe so - I checked empirically, and also see https://github.com/openshift/vagrant-openshift/blob/master/lib/vagrant-openshift/action/install_origin.rb#L43 since a long time ago

(It's a bit grotty that the "which" is required in "sudo `which openshift`" - this is because secure_path in /etc/sudoers does not include the openshift binary path - hence the suggestion for newbies that they do sudo service openshift start instead)